hibernate-osgi/src/main/java/org/hibernate/osgi/OsgiClassLoader.java

```diff
@@ -212,4 +214,18 @@ public void clear() {
 		resourceCache.clear();
 	}
 
+	public void removeBundle(Bundle bundle) {
```

Comment:
Actually, taking back part of what I said -- removing `bundles` entirely cleans up logic elsewhere...